common/scripted_effects/hfe_effect.txt

```diff
@@ -1729,12 +1790,17 @@ hfe_effect_gov_fix_to_hive = {
 				ethic = ethic_gestalt_consciousness
 			}
 			authority = auth_machine_intelligence
+			origin = origin_default
+			civics = {
```

Comment:
not necessary

Reply:
Wrong. Wrong. Wrong. The Error Logs beg to differ.

Reply:
What logs?

Reply:
Paradox's official Error Logs. If you don't have the Civics and an Origin, the game goes crazy and it sometimes won't even start.

Reply:
Okay, i will check it by myself later